docs/OTLabels/validation/modelvalidation.md

```diff
@@ -0,0 +1,234 @@
+Such case can be seen in scene 3 at nighttime where cars have been parked on the sidewalk in the following image:
+
+![Scene 3 Night Parked Cars](assets/scenes/scene_3_night_parked_cars.jpg)
+
+Thus, such case could influence the values of the calculated metrics for better or worse.
+
+#### Scene 4
+
+Scene 4 captures a rural road.
+
+| Scene 4                                                                        | Scene 5                                                                        |
+| ------------------------------------------------------------------------------ | ------------------------------------------------------------------------------ |
+| ![Scene 4](assets/scenes/scene_4.jpg)                                          | ![Scene 5](assets/scenes/scene_5.jpg)                                          |
+| ![Scene 4 mAP](assets/plots/scene_4/mAP_diagram_scene_4.png)                   | ![Scene 5 mAP](assets/plots/scene_5/mAP_diagram_scene_5.png)                   |
+| ![Scene 4 TIDE](assets/plots/scene_4/tide_errors_diagram_scene_4.png)          | ![Scene 5 TIDE](assets/plots/scene_5/tide_errors_diagram_scene_5.png)          |
+| ![Scene 4 TIDE Stacked](assets/plots/scene_4/tide_stacked_diagram_scene_4.png) | ![Scene 5 TIDE Stacked](assets/plots/scene_5/tide_stacked_diagram_scene_5.png) |
+
+The **Missed Ground Truth Error** (Miss) is pretty high for the YOLOv5s and the YOLOv5x models with an error value over 50% compared to the YOLOv5m and YOLOv5l models.
+At this moment, we could not come up with an explanation of why the YOLOv5x model's missed ground truth error is that much higher than its `m` and `l` counterpart.
+But there is an explanation why the missed ground truth error is high across all models.
+The reason lies in the dataset of scene 4 itself:
+
+| Scene 4 - images with cut off objects                                  |                                                                            |
+| ---------------------------------------------------------------------- | -------------------------------------------------------------------------- |
+| ![Scene 4 Person cut off 1](assets/scenes/scene_4_cutoff_person_1.jpg) | ![Scene 4 Person cut off 2](assets/scenes/scene_4_cutoff_person_2.jpg)     |
+| ![Scene 4 Person cut off 3](assets/scenes/scene_4_cutoff_person_3.jpg) | ![Scene 4 Person cut off](assets/scenes/scene_4_cutoff_car_1.jpg)          |
+| ![Scene 4 Person cut off](assets/scenes/scene_4_cutoff_car_2.jpg)      | ![Scene 4 Truck cut off](assets/scenes/scene_4_cutoff_truck.jpg)           |
+
+The images above contain objects, which are surrounded by red bounding boxes, that are only partly in their respective image.
+Therefore, a model might not have been able to detect these objects due to a low confidence score and thus resulting in misses.
+These type of images appear often in the dataset.
+Another thing to keep in mind is that there is not much traffic on a rural road.
```

Comment:
"(...) on this rural road." ?

Reply:
Fixed in the latest commits that I pushed :)